src/main/resources/org/jenkins/plugins/lockableresources/LockStep/config.jelly

```diff
@@ -19,6 +19,9 @@
   <f:entry field="skipIfLocked">
     <f:checkbox title="${%entry.inversePrecedence.skipIfLocked.title}"/>
   </f:entry>
+  <f:entry title="${%entry.resourceSelectStrategy.title}" field="resourceSelectStrategy">
```

Comment:
combo-box will be fine
![image](https://user-images.githubusercontent.com/89339813/212177545-37a5f72a-5415-46fe-a9fc-a10141235352.png)

Reply:
Script generator does not work 
![image](https://user-images.githubusercontent.com/89339813/212178369-76616707-9be9-433a-8036-47c0a3ca3567.png)

Reply:
Where can I find this page?

Reply:
> Where can I find this page?

[/pipeline-syntax/](http://localhost:8080/jenkins/pipeline-syntax/)